src/main/scala/HeaderEnum.scala

```diff
@@ -3,6 +3,7 @@ package testchipip
 import chisel3._
 import chisel3.util.log2Up
 import scala.collection.mutable.{HashMap, ListBuffer}
+import scala.language.postfixOps
```

Comment:
This quiets down the following warning but I'm not sure if it is the 'correct' thing to do:
```
[warn] target-design/chipyard/generators/testchipip/src/main/scala/HeaderEnum.scala:10:103: postfix operator mkString should be enabled
[warn] by making the implicit value scala.language.postfixOps visible.
[warn] This can be achieved by adding the import clause 'import scala.language.postfixOps'
[warn] or by setting the compiler option -language:postfixOps.
[warn] See the Scaladoc for value scala.language.postfixOps for a discussion
[warn] why the feature should be explicitly enabled.
[warn]     h.toSeq.sortBy(_._2).map { case (n,i) => s"#define ${prefix.toUpperCase}_${n.toUpperCase} $i\n" } mkString
[warn]    
```